src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Initialization.cs

```diff
@@ -33,6 +56,16 @@ static libcrypto()
             ERR_load_crypto_strings();
         }
 
+        internal static void Initialize()
+        {
```

Comment:
I'd suggest adding a short comment here that this is a temporary function, meant to be a nop just to give multiple other static cctors a hook to ensure CryptoInitializer's static cctor is invoked.